<?xml version="1.0" encoding="UTF-8"?>
<!--Copyright 2024 The MathWorks, Inc.-->

<rsccat xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.0" locale="ko_KR" product="parallel" xsi:noNamespaceSchemaLocation="../../resources/schema/msgcat.xsd">
  <message>
    <entry key="AddAdditionalPathsToPath">Error encountered while adding AdditionalPaths to MATLAB path.</entry>
    <entry key="AddAttachedFilesToPath">AttachedFiles를 MATLAB 경로에 추가하는 중 오류가 발생했습니다.</entry>
    <entry key="AddAttachedFilesToPathZipError">AttachedFiles의 압축을 푸는 중 오류가 발생했습니다. 이 문제는 AttachedFiles의 크기가 4GB보다 큰 경우에 발생할 수 있습니다.</entry>
    <entry key="AutoAttachedFiles">''{0}''에 자동으로 추가된 파일:\n{1}</entry>
    <entry key="CannotLoadFromMatFile" context="error">task 객체는 MAT 파일에서 불러오거나 생성자에서 직접 만들 수 없습니다.</entry>
    <entry key="DiaryIncomplete">태스크가 아직 실행 중이므로 이 태스크의 diary가 불완전할 수 있습니다.</entry>
    <entry key="EvaluateTaskUnexpectedError">dctEvaluateTask/nEvaluateTask에서 예기치 않은 오류가 발생했습니다.</entry>
    <entry key="GetInputData">태스크에서 입력 데이터를 가져오는 중 오류가 발생했습니다.</entry>
    <entry key="InvalidDeleteSyntax">Invalid syntax. Use "delete(task)" to delete a task.</entry>
    <entry key="MJSDataStorageFailed">태스크 데이터를 데이터베이스에서 읽기 또는 쓰기에 실패했습니다.</entry>
    <entry key="MJSTaskNotFound">태스크를 찾을 수 없습니다.</entry>
    <entry key="MJSUnableToSubmitResult">결과를 jobmanager에 반환할 수 없습니다. 이유는 다음과 같습니다.\n{0}</entry>
    <entry key="NoAutoAttachedFiles">''{0}''에 자동으로 추가된 파일이 없습니다.</entry>
    <entry key="PostTaskUnexpectedError">dctEvaluateTask/nPostTaskFinish에서 예기치 않은 오류가 발생했습니다.</entry>
    <entry key="PropertySetVersionMismatch">Properties must be set on tasks in the same MATLAB release that created the task: {0}</entry>
    <entry key="TaskCancellationFailed">클러스터가 태스크 실행을 취소하지 못했습니다. 오류: {0}</entry>
    <entry key="TaskDestructionFailed">클러스터가 태스크를 올바르게 제거하지 못했습니다. 오류: {0}</entry>
    <entry key="TaskFinishUserError">사용자가 제공한 taskFinish 함수 ''{0}''에서 오류가 발생했습니다.</entry>
    <entry key="TaskResultTooLarge">태스크 결과가 너무 커서 저장할 수 없습니다.</entry>
    <entry key="TaskStartupUserError">사용자가 제공한 taskStartup 함수 ''{0}''에서 오류가 발생했습니다.</entry>
    <entry key="UserCancellation">The task was cancelled by user "{0}" on machine "{1}".</entry>
    <entry key="UserCancellationWithMessage">The task was cancelled by user "{0}" on machine "{1}" with message: "{2}".</entry>
    <entry key="WarningLimitReached">최대 경고 수에 도달했습니다. 추가 경고는 무시됩니다.</entry>
  </message>
</rsccat>
